docs/package.json

```diff
@@ -13,6 +13,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "markdown-it": ">=12.3.2", 
```

Comment:
I asked this morning AiB if they could bump the dependency of markdown-it in vuepress-theme-cosmos, because I thought that's the dependency from which we get it. If we add markdown-it explicitly as a dependency, will it override the version used in vuepress-theme-cosmos?

Reply:
Good question. I'm not sure

Reply:
Looks like it's possible [like this using yarn](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/) or [like this using npm-shrinkwrap]()https://nodejs.org/en/blog/npm/managing-node-js-dependencies-with-shrinkwrap/).